### PR TITLE
added Dockerfile and readme how to use in docker

### PR DESCRIPTION
**Pull request by kevit**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,3 +32,9 @@ Debian and RPM packages can be found on the [releases][3] page.
 [1]: https://ip-ranges.amazonaws.com/ip-ranges.json
 [2]: https://docs.aws.amazon.com/general/latest/gr/aws-ip-ranges.html
 [3]: https://github.com/sampointer/digaws/releases/
+
+### Docker
+
+git clone
+docker build -t digaws .
+docker run --rm -it digaws /app/main $(dig netflix.com +short)
```
Comment (sampointer) [inline README.md:40]: I'd like to see this within a ```bash block, consistent with the rest of the README.

**Review by sampointer — CHANGES_REQUESTED** [review]
Building the image in this way produces a 1.28GB final image (although admittedly 839MB of that is `golang:latest` and so some may be able to take advantage of layer caching, at least until the image is outdated).

A `Dockerfile` built something like this:

```
FROM golang:latest AS build
RUN apt-get update && apt-get install ca-certificates -yq
COPY . /go/src/
WORKDIR /go/src
RUN go test && go install

FROM scratch
COPY --from=build /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
COPY --from=build /go/bin/digaws ./digaws
ENTRYPOINT [ "./digaws" ]
```

produces a 12MB image. I'm not happy with merging the former both for this reason and that a full distro image and compiler toolchain also places a burden on those who scan their repositories for vulnerabilities.

File: README.md (hunk 1 above)
Comment (kevit) [reply README.md]: updated

```
REPOSITORY              TAG                 IMAGE ID            CREATED              SIZE
digaws                  latest              a2767eda8765        57 seconds ago       12.9MB

```

**Review by sampointer — APPROVED** [review]

**Comment by sampointer** [comment]
Many thanks for your contribution :+1:

**Comment by sampointer** [comment]
I was wondering, for my own interest, what your use-case for this was? If you wanted to support for another `GOOS` for example that would interest me as I could produce native binaries for it. The utility of dockerizing this isn't obvious and I'm interested!

**Comment by kevit** [comment]
I use quite often dockerized cli tools, f.e https://hub.docker.com/r/presbrey/mylg or https://hub.docker.com/r/sjourdan/toolbox or even https://hub.docker.com/r/harisekhon/nagios-plugins. Reason: canned and may be used ad-hoc, I do consulting and environments all the time new for me

**Pull request merged**